plugin/pkg/scheduler/factory/factory.go

```diff
@@ -285,6 +285,10 @@ func (factory *ConfigFactory) makeDefaultErrorFunc(backoff *podBackoff, podQueue
 	}
 }
 
+func (factory *ConfigFactory) SetBindPodsRateLimiter(rateLimiter util.RateLimiter) {
```

Comment:
In line 43 there are two constants defined: BindPodsQps and BindPodsBurst.
They seem to be unneded now (i.e. these are default values for flags now). We should get rid of them and either just remove setting rate limitter in NewConfigFactory or pass the values of the flag there.